gems/pending/spec/appliance_console/database_replication_spec.rb

```diff
@@ -0,0 +1,167 @@
+require "appliance_console/logging"
+require "appliance_console/prompts"
+require "appliance_console/database_replication"
+require "appliance_console/logging"
+require "tempfile"
+
+describe ApplianceConsole::DatabaseReplication do
+  before do
+    allow(subject).to receive(:say)
+    allow(subject).to receive(:clear_screen)
+    allow(subject).to receive(:agree)
+    allow(subject).to receive(:ask_for_ip_or_hostname)
+    allow(subject).to receive(:ask_for_password_or_none)
+    allow(subject).to receive(:ask_for_password)
+  end
+
+  context "#ask_for_unique_cluster_node_number" do
+    it "should ask for a unique number" do
+      expect(subject).to receive(:ask_for_integer).with(/uniquely identifying this node/i).and_return(1)
+
+      subject.ask_for_unique_cluster_node_number
```

Comment:
I feel like the important expectation here is missing.

We should really care more about whether the correct value was set on the object than whether the question was asked correctly.